docs/spec_draft.md

```diff
@@ -1185,6 +1187,43 @@ output of `true_branch` is returned, else if pred is `false`, output of
 
 [Back to Ops](#index-of-ops)
 
+## stablehlo.imag
+
+### Semantics
+
+Extracts an imaginary value element-wise from `operand` and produces a `result`
+tensor.
```

Comment:
Extracts the imaginary part, element-wise, from the `operand` and produces a `result` tensor.